README.md

```diff
@@ -2,7 +2,7 @@
 
 ## About
 
-This software provides conversion between [hdf5 files](https://www.hdfgroup.org/HDF5/) marked up in [openPMD](https://github.com/openPMD/openPMD-standard)-conforming way and the [GDF file format](http://www.pulsar.nl/gpt/).
+This software provides conversion between [openPMD-format file](https://github.com/openPMD/openPMD-standard)  and the [GDF file format](http://www.pulsar.nl/gpt/).
```

Comment:
I think it's fine to use `openPMD file` instead of the weird and lengthy `openPMD-format file`. Optionally with a clarification somewhere around here.